### PR TITLE
Chore/detox

### PR DESCRIPTION
**Pull request by robinheinze**
Just a few chores while things are fresh to maybe help set us up for next year a little better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -65,3 +65,69 @@ If you have to bypass lint for a special commit that you will come back and clea
...
+
+# Detox End-To-End Testing
+
+## Setup
+
+To get your Detox tests up and running, you'll need to install some global dependencies:
+
+1. Install the latest version of [Homebrew](https://brew.sh/)
+
+This conversation was marked as resolved by robinheinze
```
Comment (jamonholmgren) [inline README.md]: What's this?

File: README.md [hunk 2]
```diff
@@ -65,3 +65,69 @@ If you have to bypass lint for a special commit that you will come back and clea
 **Understanding Linting Errors**
 
 The linting rules are from tslint-config-prettier. [Regular TS errors can be found with descriptions here](https://palantir.github.io/tslint/rules/).
+
+# Detox End-To-End Testing
+
+## Setup
+
+To get your Detox tests up and running, you'll need to install some global dependencies:
```
Comment (jamonholmgren) [inline README.md]:
```suggestion
_Note that Detox is only configured for macOS._

  To get your Detox tests up and running, you'll need to install some global dependencies:
```

File: README.md [hunk 3]
```diff
@@ -65,3 +65,69 @@ If you have to bypass lint for a special commit that you will come back and clea
...
+1. Install the latest version of [Homebrew](https://brew.sh/)
+
+This conversation was marked as resolved by robinheinze
+
+2. Make sure you have Node installed (at least 8.6.0). If you don't:
+
+If you use NVM:
+
+```bash
+nvm install node
```
Comment (jamonholmgren) [inline README.md]: Just a minor, low gafo comment, but I don't think we should bother showing `nvm` install instructions. If they have `nvm` installed, they know how to install Node. Just do the Homebrew one.

IMO.

File: README.md [hunk 4]
```diff
@@ -65,3 +65,69 @@ If you have to bypass lint for a special commit that you will come back and clea
...
+```
+
+Or if you'd prefer to install directly from Homebrew
+
+```bash
+brew update && brew install node
+```
+
+3. Install `applesimutils, which will allow Detox to communicate with the iOS simulator:
```
Comment (jamonholmgren) [inline README.md]:
```suggestion
3. Install `applesimutils`, which will allow Detox to communicate with the iOS simulator:
```

File: README.md (hunk 1 above)
Comment (robinheinze) [reply README.md]: HA. Copy paste error. I copied this from the Bowser PR I did for Detox a while back 🤦‍♀ Thanks for catching.

**Pull request merged**